src/io/flutter/bazel/Workspace.java

```diff
@@ -33,6 +33,8 @@
 public class Workspace {
   private static final String PLUGIN_CONFIG_PATH = "dart/config/intellij-plugins/flutter.json";
 
+  public static final String BAZEL_URI_SCHEME = "google3://";
```

Comment:
@jakemac53 please let me know if you clean this up and just make the scheme bazel://

Reply:
Perhaps add a todo here that we anticipate a future change to this url scheme?